sdk/vision/Azure.AI.Vision.ImageAnalysis/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0 (2024-10-18)
+General availability release.
```

Comment:
In Azure SDK terms this is called a "Stable release" ("GA" term is for REST APIs).